### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by MrHelp**
Work with Java 11. Add getFees.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.1</version>
+        </dependency>
```
Comment (qinshaohui) [inline pom.xml]: I don't think this is needed anymore after https://github.com/binance-chain/java-sdk/pull/3/files.

**Comment by qinshaohui** [comment]
Thanks for this. Please remove the jaxb-api dependency and I will merge.

File: src/main/java/com/binance/dex/api/client/domain/broadcast/TransactionOption.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 /**
- * Optional fields for Bianace DEX standard transaction
+ * Optional fields for Bianace dex standard transaction
```
Comment (yutianwu) [inline src/main/java/com/binance/dex/api/client/domain/broadcast/TransactionOption.java]: typo here, Bianace

**Comment by MrHelp** [comment]
done

**Pull request merged**